pkg/socketenricher/tracer.go

```diff
@@ -53,7 +53,7 @@ func init() {
 
 const (
 	SocketsMapName          = "gadget_sockets"
-	SocketsValueName        = "sockets_value"
+	SocketsValueName        = "gadget_socket_value"
 	optionalFieldsStartName = "optional_fields_start"
```

Comment:
Shouldn't it be `gadget_optional_fields_start` too?

Reply:
It's a field inside the structure, so it's already namespaced by the structure itself.